test/integration/features/basic.feature

```diff
@@ -124,7 +124,7 @@ Feature: Basic test
 
     @darwin @linux @windows
     Scenario: CRC status check
-        When with up to "2" retries with wait period of "1m" command "crc status --log-level debug" output should not contain "Running"
+        When with up to "8" retries with wait period of "2m" command "crc status --log-level debug" output should not contain "Running"
```

Comment:
@praveenkumar , this status check comes after `crc stop -f`, so I don't think 2min waits are necessary, I'd even go down to 10x10s or something. After `stop -f`, the status immediately shows `Stopped` state (I'd keep a few retries for the robustness of the test, but not much more). Of course, checking status after `crc start` is a different story!